cqproto/grpc.go

```diff
@@ -195,11 +195,11 @@ func tablesFromProto(in map[string]*internal.Table) map[string]*schema.Table {
 func tableFromProto(v *internal.Table) *schema.Table {
 	cols := make([]schema.Column, len(v.GetColumns()))
 	for i, c := range v.GetColumns() {
-		cols[i] = schema.Column{
+		cols[i] = schema.SetColumnMeta(schema.Column{
```

Comment:
Looks convoluted,  maybe there's a simpler/cleaner way to convert between `internal.Column` and `schema.Column`? a method on `intenral.Column` that gives you `schema.Column` maybe? If import cycles won't be an issue that is

Reply:
internal.Column is auto generated